### PR TITLE
Move GinjFo.com.xml into src/chrome/content/rules

### PR DESCRIPTION
**Pull request by koops76**
#11756
How could you miss that, @J0WI? 😁

### CODE REVIEW

**Comment by koops76** [comment]
Both `^` and `www` have incomplete certificate chain. I guess it should be deleted.

**Comment by koops76** [comment]
@J0WI I disabled this ruleset by default.

File: src/chrome/content/rules/GinjFo.com.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		^
 		openx.ginjfo.com
 -->
-<ruleset name="GinjFo.com">
+<ruleset name="GinjFo.com" default_off="mismatch">
```
Comment (J0WI) [inline src/chrome/content/rules/GinjFo.com.xml]: Please use `cert-chain`

File: src/chrome/content/rules/GinjFo.com.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 		^
```
Comment (J0WI) [inline src/chrome/content/rules/GinjFo.com.xml]: `^` is mismatched

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/GinjFo.com.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-		^
-		openx.ginjfo.com
+	Mismatched:
+		- ^
 -->
-<ruleset name="GinjFo.com" default_off="mismatch">
+
+<ruleset name="GinjFo.com" default_off="cert-chain">
 	<target host="ginjfo.com" />
 	<target host="www.ginjfo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/GinjFo.com.xml]: `openx.ginjfo.com`

**Comment by J0WI** [comment]
Thanks for fixing this

**Pull request merged**